app/Http/Controllers/V2/Dashboard/ActiveProjectsTableController.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace App\Http\Controllers\V2\Dashboard;
+
+use App\Helpers\TerrafundDashboardQueryHelper;
+use App\Http\Controllers\Controller;
+use App\Models\V2\Forms\FormOptionList;
+use App\Models\V2\Forms\FormOptionListOption;
+use App\Models\V2\Projects\Project;
+use App\Models\V2\Projects\ProjectReport;
+use App\Models\V2\Sites\SiteReport;
+use App\Models\V2\TreeSpecies\TreeSpecies;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class ActiveProjectsTableController extends Controller
+{
+    public function __invoke(Request $request)
+    {
+        return response()->json([
+            'project_list' => $this->getAllProjects($request),
+        ]);
```

Comment:
I believe the normal pattern here would be to have a resource class like the ProjectReportResource, and let it handle including the correct items for the API response.